### ISSUE TITLE
Button shows after delete 

### ISSUE THREAD

**Issue by emerg** [eCollect/vue-swipe-actions#4]
Thank you for such a great component.
I have a slight issue, and I was not able to see anything regarding this in the docs.
When I swipe to reveal the actions, one of my actions is to delete that item from the array. When I delete the item, the next item below now already has the buttons/actions revealed for it, without swiping. I would like only for the delete button to be shown when an item is swiped on. 
Thank you for the help.

<img width="353" alt="screen shot 2018-11-05 at 12 39 07 pm" src="https://user-images.githubusercontent.com/4573650/48015888-2742ca80-e0f8-11e8-8278-126e4f640e22.png">
<img width="340" alt="screen shot 2018-11-05 at 12 39 17 pm" src="https://user-images.githubusercontent.com/4573650/48015895-2b6ee800-e0f8-11e8-8a7c-bbf17664cc67.png">
<img width="337" alt="screen shot 2018-11-05 at 12 39 29 pm" src="https://user-images.githubusercontent.com/4573650/48015903-2e69d880-e0f8-11e8-977a-cca14c8bfdba.png">

**Comment by nanov** [comment]
Hi!

Are you using the ```SwipeList``` or the ```SwipeOut``` component?

If using ```SwipeList``` are you setting a ```transition-key``` attribute?  (some sort of unique attribute/value in the item). 

When that attributed is not supplied the component uses the array index as key, and when removing some item the next one gets its index and therefore its state.

For example the objects in your array are in the following format :
```js
{
  id: 124566,
  title: 'harum...',
  timestamp: 2357794,
  // ...
}
```

You should supply ```id``` ( string, ie. the property name not value ) as ```transition-key```.

**Comment by nanov** [comment]
An item may have an id regardless of its origin ( ids are not only db generated ).

The ```transition-key``` property is a sugar for [vue's ```key``` property](https://vuejs.org/v2/guide/list.html#key).

One thing you could do is to generate an id when generating the item ( an [uuid/v4](https://www.npmjs.com/package/uuid) for example ). In other words, depending on your carts logic either in a vuex action/mutation or whenever the item is added to your cart.

**Comment by emerg** [comment]
Thank you for the help Dimitar. Yes I went ahead and tried generating an ID on the creation of the cart objects so that I could then pass that into the transition-key property, and it is working as expected. Thank you for you help and for putting together a great vue component!

**Comment by emerg** [comment]
Actually, still having this issue, even though passing in a unique ID now. The difference is that the issue is reversed. instead of it happening to an item below, it now happens to the item above. Is there there any function I can call to simply re-initialize the SwipeList component?

**Comment by nanov** [comment]
That is rather strange. I will take a look now to see what exactly happens.
A gist/repository reproducing the problem would come in great help.

**Comment by nanov** [comment]
I am sorry but I cannot reproduce the problem, are you sure your ids are unique? Are your sure your are passing the right key name ( a limitation is that the property cannot be nested, ie. ```meta.id``` won't work but ```id``` would ) ?

I've just made a release which has the ```close``` method passed to action slots as well, additionally I've updated the [demo](https://github.com/eCollect/vue-swipe-actions/blob/master/src/App.vue) to show a situation like yours ( removing an item trough the actions ).

For more assistance i would need a test case reproducing the problem.
